cypress/integration/MenuItem/Is_clickable.feature

```diff
@@ -1,6 +1,6 @@
 Feature: The MenuItem has an onClick api
 
     Scenario: The user clicks on the MenuItem
-        Given a MenuItem with onClick handler is rendered
+        Given a MenuItem with onClick handler and value is rendered
         When the MenuItem is clicked
-        Then the onClick handler is called
+        Then the onClick handler is called with value
```

Comment:
IMO, the most complex part of the Menu is it's ability to include submenus that pop-out. This isn't included yet. And I believe it's currently broken functionality too. So I guess you can ignore this, and I will add tests for that when I work on the Menu (make it powered by the Popper)

Reply:
Yeah. I tried testing this, but you're right, it seems to be broken currently.

Reply:
I'll resolve this one too. I'll deal with this later